packages/runtime/src/widgets/Charts/StackBarChart.tsx

```diff
@@ -80,6 +81,7 @@ export const StackBarChart: React.FC<BarChartProps> = (props) => {
           },
         },
       }}
+      plugins={[ChartDataLabels]}
```

Comment:
Any reason not to add this to every Chart?

Reply:
i ll have to test it thoroughly as even for Line Chart if no configuration is provided it just explodes the UI and it looks weird. I ll check their docs so that we can have properties users can use to switch it on and off.

Reply:
Okay we'll just keep it for the stacked bar chart for now